config/overlays/test/configmap/inferenceservice.yaml

```diff
@@ -80,6 +80,14 @@ data:
             "multiModelServer": false
           }
         },
+        "paddle": {
+            "image": "ruminateer/paddleserver",
+            "defaultImageVersion": "latest",
```

Comment:
@Ruminateer For now you can pin to a specific tag, in e2e script we replace latest with the PULL_BASED_SHA which is different for each run.